docs/content/https/tls.md

```diff
@@ -159,6 +159,75 @@ data:
 
 If no default certificate is provided, Traefik generates and uses a self-signed certificate.
 
+### ACME Default Certificate
+
+You can configure Traefik to use an ACME provider (like Let's Encrypt) to generate the default certificate.
+The configuration to resolve the default certificate should be defined in a TLS store:
+
+!!! important "Precedence with the `defaultCertificate` option"
+
+    The `defaultCertificate` definition takes precedence on the ACME default certificate configuration.
```

Comment:
```suggestion
    The `defaultCertificate` definition takes precedence over the ACME default certificate configuration.
```

I don't think this is quite clear as it is, though, especially if it's important?